### PR TITLE
Add test for UDashboard

### PR DESCRIPTION
**Pull request by xitronix**
after #1293

### CODE REVIEW

**Comment by marekkirejczyk** [comment]
Needs rebase

File: universal-login-react/test/config/setupGlobalFetch.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+(global as any).fetch = require('node-fetch');
```
Comment (JustynaBroniszewska) [inline universal-login-react/test/config/setupGlobalFetch.ts]: It looks pretty similar to [this](https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-wallet/test/integration/helpers/setupCryptoCompare.ts). Maybe we should merge it?

Comment (marekkirejczyk) [reply universal-login-react/test/config/setupGlobalFetch.ts]: extract setupGlobalFetch and deduplicate

File: universal-login-react/test/ui/UDashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  let contractAddress: string;
+  let privateKey: string;
+  const ensName = `jarek.mylogin.eth`;
+  const initialAmount = `1.99`;
+
+  beforeEach(async () => {
+    ([wallet] = await getWallets(createMockProvider()));
+    const {sdk} = await setupSdk(wallet, '33113');
+    sdk.priceObserver.getCurrentPrices = () => {
+      return Promise.resolve({ETH: {USD: 1, DAI: 1, ETH: 1}});
```
Comment (marekkirejczyk) [inline universal-login-react/test/ui/UDashboard.tsx]: let's make it a bit more intuitive, e.g.:
`return Promise.resolve({ETH: {USD: 100, DAI: 99, ETH: 1}});`

File: universal-login-react/test/ui/UDashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    sdk.start();
+    ({contractAddress, privateKey} = await createWallet(ensName, sdk, wallet));
+    deployedWallet = new DeployedWallet(contractAddress, ensName, privateKey, sdk);
+  });
+
+  it('update usd balance amount', async () => {
+    const appWrapper = mount(<UDashboard deployedWallet={deployedWallet} />);
+    const uButton = appWrapper.find('.udashboard-logo-btn');
+    uButton.simulate('click');
+    const balanceComponent = appWrapper.find('.universal-login-balance-amount');
```
Comment (cehali) [inline universal-login-react/test/ui/UDashboard.tsx]: Extract to separate file e.g. UDashboardPage

File: universal-login-wallet/test/integration/ui/Transfer.tsx [hunk 4]
```diff
@@ -4,13 +4,13 @@ import {providers, utils, Contract} from 'ethers';
...
 import {deployMockToken} from '@universal-login/commons/testutils';
+import {setupGlobalFetch} from '@universal-login/react/testutils';
 import {setupSdk} from '@universal-login/sdk/testutils';
 import {Services} from '../../../src/ui/createServices';
 import {setupUI} from '../helpers/setupUI';
-import {setupCryptoCompare} from '../helpers/setupCryptoCompare';
 import {AppPage} from '../pages/AppPage';
 
-setupCryptoCompare();
+setupGlobalFetch();
```
Comment (JustynaBroniszewska) [inline universal-login-wallet/test/integration/ui/Transfer.tsx]: move it to `registrars.ts`

**Pull request merged**